wiktextract/data/fr/zh_pron_tags.json

```diff
@@ -0,0 +1,3635 @@
+{
```

Comment:
You don't have to copy the file from the `en` folder. You could use an empty dictionary `{}` in this file.